mealie/core/security/security.py

```diff
@@ -54,57 +54,77 @@ def user_from_ldap(db: AllRepositories, username: str, password: str) -> Private
 
     if settings.LDAP_TLS_INSECURE:
         ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)
+
     ldap.set_option(ldap.OPT_REFERRALS, 0)
     ldap.set_option(ldap.OPT_PROTOCOL_VERSION, 3)
     conn = ldap.initialize(settings.LDAP_SERVER_URL)
 
     if settings.LDAP_TLS_CACERTFILE:
         conn.set_option(ldap.OPT_X_TLS_CACERTFILE, settings.LDAP_TLS_CACERTFILE)
         conn.set_option(ldap.OPT_X_TLS_NEWCTX, 0)
-    user = db.users.get_one(username, "email", any_case=True)
 
-    if not settings.LDAP_BIND_TEMPLATE:
-        return False
+    user = db.users.get_one(username, "email", any_case=True)
 
     if not user:
-        user_bind = settings.LDAP_BIND_TEMPLATE.format(username)
+        user_bind = "cn={}, {}".format(username, settings.LDAP_BASE_DN)
```

Comment:
This might break some LDAP setups, I suggest to leave the `LDAP_BIND_TEMPLATE` as it is.
For example on my server I need `uid={},...`, not `cn={},...`.

Reply:
Did you get LDAP  to work with v1? It seems that you are using FreeIPA.
Mine was working with v0.5, but v1 broke it.
I am trying

```
LDAP_BIND_TEMPLATE=uid={},cn=users,cn=accounts,dc=...,dc=...
LDAP_BASE_DN=cn=users,cn=accounts,dc=...,dc=...
```

Reply:
I'm not using FreeIPA, I use my own custom setup with OpenLDAP. But yes, it worked with v1.